src/objective-c/tests/InteropTests/InteropTests.m

```diff
@@ -701,21 +703,21 @@ - (void)testResponsesOver4MBAreAcceptedIfOptedIn {
   XCTAssertNotNil([[self class] host]);
   __weak XCTestExpectation *expectation =
       [self expectationWithDescription:@"HigherResponseSizeLimit"];
+  __block NSError *callError = nil;
 
   RMTSimpleRequest *request = [RMTSimpleRequest message];
   const size_t kPayloadSize = 5 * 1024 * 1024;  // 5MB
   request.responseSize = kPayloadSize;
 
   [GRPCCall setResponseSizeLimit:6 * 1024 * 1024 forHost:[[self class] host]];
-
   [_service unaryCallWithRequest:request
                          handler:^(RMTSimpleResponse *response, NSError *error) {
-                           XCTAssertNil(error, @"Finished with unexpected error: %@", error);
-                           XCTAssertEqual(response.payload.body.length, kPayloadSize);
```

Comment:
Don't we need to move this assert to after waitForExpectationsWithTimeout, similar to callError?

Reply:
The purpose of this test is to test message size filter not get triggered, so this assert is unrelated (the same minimizing test concept).